src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/SqlImportOperation.cs

```diff
@@ -87,7 +87,7 @@ public IReadOnlyList<(Table table, Index index, bool pageCompression)> UniqueInd
 
             if (_schemaInformation.Current >= SchemaVersionConstants.RenamedIndexForResourceTable)
             {
-                list.Add((VLatest.Resource, VLatest.Resource.IX_Resource_ResourceSurrogateId, false));
+                ////list.Add((VLatest.Resource, VLatest.Resource.IX_Resource_ResourceSurrogateId));
             }
```

Comment:
I question is  why isn't the logic something like:
```csharp
  if (_schemaInformation.Current >= SchemaVersionConstants.RenamedIndexForResourceTable && _schemaInformation.Current < 34 /*SchemaVersionConstants.DroppedResourceSurrogateIX*/)
  {
      list.Add((VLatest.Resource, VLatest.Resource.IX_Resource_ResourceSurrogateId));
  }
```

Reply:
Because we don't care about how this is handled. As long as it is not broken.

Reply:
Ah got it, all good